draft-ietf-anima-brski-prm.md

```diff
@@ -1103,589 +1166,583 @@ pledgeenrollrequesttrigger = {
 
-To support Registrar-Agents requesting a signature wrapped CA certificate(s) object, a new endpoint for BRSKI-PRM is defined on the registrar: "/.well-known/brski/wrappedcacerts"
+### Request (no artifact)
 
-The Registrar-Agent SHALL requests the EST CA trust anchor database information (in form of CA certificates) by HTTP GET.
+In this exchange, the request is a result of the HTTP(S) default transport for this specification.
+There is no artifact provided to the registrar.
 
+### Response Artifact: CA-Certificates (caCerts) {#cacerts-artifact}
 
-### Response Artifact: cACert-Response (cACert-Resp)
+The CA-Certificates (caCerts) artifact SHALL be an authenticated self-contained object signed by the registrar, containing the domain trust anchors, i.e., CA (root or issuing) certificate(s).
 
-The Content-Type header of the response SHALL be: `application/jose+json`.
+For the JWS-signed JSON format used by this specification, the caCerts artifact MUST use the "General JWS JSON Serialization Syntax" defined in {{Section 7.2.1 of !RFC7515}}, which MUST contain the JSON CA Data defined in {{cacerts-data}} in the JWS Payload.
 
-This is a deviation from the Content-Type header values used in EST {{RFC7030}} and results in additional processing at the domain registrar (as EST server).
-The additional processing is to sign the CA certificate(s) information using the registrar LDevID credentials.
-This results in a signed CA certificate(s) object (JSON-in-JWS), the CA certificates are provided as base64-encoded "x5bag" (see definition in {{RFC9360}}) in the JWS payload.
+{{cacerts_representation}} summarizes the serialization of the JWS-signed JSON caCerts artifact:
 
 ~~~~
-# The CA certificates data with registrar signature in 
-# General JWS Serialization syntax
 {
-  "payload": BASE64URL(certs),
+  "payload": BASE64URL(UTF8(JSON CA Data)),
   "signatures": [
     {
       "protected": BASE64URL(UTF8(JWS Protected Header)),
       "signature": BASE64URL(JWS Signature)
     }
   ]
 }
+~~~~
+{: #cacerts_representation title='Voucher\' Representation in General JWS JSON Serialization Syntax' artwork-align="left"}
+
+The JSON CA Data MUST be UTF-8 encoded to become the octet-based JWS Payload defined in {{RFC7515}}.
+The JWS Payload is further base64url-encoded to become the string value of the `payload` member as described in {{Section 3.2 of RFC7515}}.
+The octets of the UTF-8 representation of the JWS Protected Header are base64url-encoded to become the string value of the `protected` member.
+The generated JWS Signature is base64url-encoded to become the string value of the `signature` member.
+
+#### JSON CA Data {#cacerts-data}
+
+The JSON CA Data SHALL be a JSON document {{RFC8259}} that MUST conform with the CDDL {{!RFC8610}} data model defined in {{cacerts_CDDL_def}}:
```

Comment:
RFC9360 only defines the semantics and format for the "x5bag" field, but not the overall structure we need. Hence, we need this new CDDL.

Reply:
Okay, thanks for the clarification